src/components/text-input.components.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable default-param-last */
+/* eslint-disable unicorn/better-regex */
+import $ from 'jquery';
+
+export default class TextInput {
+    constructor(stringQuery, textEnable = false, required = true, textValid = '') {
+        this.stringQuery = stringQuery;
+
+        this.textInput = $(`${stringQuery} input[type="text"]`);
+
+        this.textValid = textValid;
+        this.textEnable = textEnable;
+        this.required = required;
+    }
+
+    getTextInput() {
```

Comment:
Same here. You class is already "Text", no need to repeat this on every method.

Reply:
done, but that wouldn't make it difficult to read to the other developers, the more specific it wouldn't be better?